R/preprocessing_io.R

```diff
@@ -633,6 +627,7 @@ process_all_manta_bedpe = function(file_df,
 #' together with the genome build (`genome_build`). A data frame will be returned with one row per file and sample IDs together with GAMBL wildcards.
 #'
 #' @param tool Name of tool.
+#' @param unix_group The unix group of the sample set.
```

Comment:
Did you manage to resolve this function? I don't see an update that makes use of the `unix_group` parameter.

Reply:
Got it, let's tackle this further down the road.